src/desktop/apps/order2/index.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import express from 'express'
+import { App } from './Components/App'
+import { renderLayout } from '@artsy/stitch'
+
+const app = (module.exports = express())
+app.set('view engine', 'jade')
+app.set('views', `${__dirname}/Components`)
+
+app.get('/order2', async (_req, res) => {
+  try {
+    const layout = await renderLayout({
+      basePath: __dirname,
+      layout: '../../components/main_layout/templates/react_index.jade',
+      config: {
+        styledComponents: true,
+      },
+      blocks: {
+        head: () => <div>head</div>,
+        body: App,
+      },
+      locals: {
+        assetPackage: 'order2',
+        ...res.locals,
+      },
+    })
+
+    res.send(layout)
+  } catch (error) {
+    console.log('(apps/order2) Error: ', error)
```

Comment:
Logging out catch'ed errors is sometimes a part of dev and sometimes what we ship - is this leftover or is there something else we should be doing here?

Reply:
Since this is a root level route wanted to log all potential errors out since any error within the entire app tree would be consumed silently if this wasn't here. For critical points I typically err on the side of required logging that points specifically to the file `(apps/order2)` rather than ENV level config